UI/window-basic-main-outputs.cpp

```diff
@@ -519,6 +536,10 @@ void SimpleOutput::Update()
 	} else if (strcmp(encoder, SIMPLE_ENCODER_NVENC) == 0) {
 		presetType = "NVENCPreset";
 
+#ifdef ENABLE_HEVC
+	} else if (strcmp(encoder, SIMPLE_ENCODER_NVENC_HEVC) == 0) {
+		presetType = "NVENCHEVCPreset";
```

Comment:
NVENC does not use different preset values for H.264, and HEVC. Just use "NVENCPreset" here.

Also, we are using the old preset values, and we eventually we want to convert to the new ones, but we can handle that in the future. See: https://github.com/FFmpeg/FFmpeg/blob/6076dbcb55d0c9b6693d1acad12a63f7268301aa/libavcodec/nvenc.c#L168

Reply:
Done.

Ok, I see.